app/models/event.rb

```diff
@@ -1,21 +1,18 @@
 # encoding: UTF-8
 class Event < ActiveRecord::Base
-  has_attached_file :image,
-    styles: { original: "800x800>", medium: "300x300>", thumb: "100x100>" },
-    path: ":rails_root/public/system/images/event/:id/:style/:filename",
-    url: "/system/images/event/:id/:style/:filename"
-  validates_attachment_content_type :image, content_type: /\Aimage\/.*\Z/
+  has_attached_file(:image,
+                    styles: { original: "800x800>",
+                              medium: "300x300>",
+                              thumb: "100x100>" },
+                    path: ":rails_root/public/system/images/event/:id/:style/:filename",
+                    url: "/system/images/event/:id/:style/:filename")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.